its/ruling/src/test/resources/expected/python-S4423.json

```diff
@@ -7,15 +7,6 @@
 174,
 213,
 ],
-'project:tornado-2.3/tornado/simple_httpclient.py':[
-210,
-],
-'project:tornado-2.3/tornado/test/httpserver_test.py':[
-110,
-115,
-120,
-125,
-],
```

Comment:
The null FQN comes from a piece of logic we have that basically does the following:
- Multiple definitions of symbol `ssl` imply we consider its FQN to be null.
- When looking at `ssl.PROTOCOL_SSLv23`, we look at the `ssl` symbol and try to see if it has any child symbol of the same name.
- If yes, we reuse that symbol, otherwise we create a new symbol, with null FQN as well.
- Before, we found the child symbol through Typeshed parsing of `ssl`, thus with the correct `ssl.PROTOCOL_SSLv23` FQN.
- Now, we don't find the child symbol because it's a variable in Typeshed, which we don't serialize.

I created SONARPY-957 to fix this.